### PR TITLE
Add an interface to merge a set of images in a single one.

### PR DESCRIPTION
**Pull request by granger35**
All is in the title.

### CODE REVIEW

File: interfaces/api/image/IImageMerger.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/**
+ * @copyright Copyright (c) 2017 B-com http://www.b-com.com/
```
Comment (jim-bcom) [inline interfaces/api/image/IImageMerger.h:2]: 2024

File: interfaces/api/image/IImageMerger.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+class XPCF_IGNORE IImageMerger :
+    virtual public org::bcom::xpcf::IComponentIntrospect {
+public:
+    IImageMerger() = default;
+	///
+    /// \brief ~IImageMerger
+	///
+    virtual ~IImageMerger() {};
+     
+	/// @brief This method converts an image source to image destination according to a given channel and color representation  layout
```
Comment (jim-bcom) [inline interfaces/api/image/IImageMerger.h:46]: Is this the right comment ?

File: interfaces/api/image/IImageMerger.h [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    /// @param[in] imgSrc: input image to convert
+    /// @param[out] imgDst: output image converted
+    /// @return FrameworkReturnCode::_SUCCESS_ id conversion succeed, else FrameworkReturnCode::_ERROR.
+    virtual FrameworkReturnCode merge(const std::vector<SRef<SolAR::datastructure::Image>> imgSrc, SRef<SolAR::datastructure::Image> & imgDst) = 0;
```
Comment (jim-bcom) [inline interfaces/api/image/IImageMerger.h:50]: Ref to vec, `const std::vector<SRef<SolAR::datastructure::Image>>&`

**Review by jim-bcom — CHANGES_REQUESTED** [review]

File: interfaces/api/image/IImageMerger.h (hunk 2 above)
Comment (granger35) [reply interfaces/api/image/IImageMerger.h]: Resolved in the last commit

File: interfaces/api/image/IImageMerger.h (hunk 3 above)
Comment (granger35) [reply interfaces/api/image/IImageMerger.h]: Resolved in the last commit

**Review by jim-bcom — APPROVED** [review]

**Pull request merged**